tests/asgi/test_boundedstream_asgi.py

```diff
@@ -154,14 +164,18 @@ async def test_iteration():
     falcon.async_to_sync(test_iteration)
 
 
-@pytest.mark.parametrize('body', [
```

Comment:
Ah, these test fixtures got a bit ugly, but I guess we'll have to live with that.
Unless there is some magic opening parenthesis out there :laughing:

Reply:
I actually don't mind them, but it may be that I'm used to them

Reply:
Me neither, it's just a massive refactoring of something that was aesthetically OK/better before.

Reply:
I guess it's inevitable when introducing a formatter that a large number of pointless changed are done.